kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/QuestionListPreview.vue

```diff
@@ -425,14 +425,6 @@
     width: 100%;
     height: 100%;
     padding: 0.5em;
-
-    &:hover {
-      background-color: white;
-    }
-
-    &.selected {
-      background-color: white;
-    }
```

Comment:
I did this in a bit of a rush 😅 - thanks for the reminder on $computedClass 😃

Reply:
No problem, can happen 🙂